packages/phone-number-privacy/combiner/test/integration/ domain.test.ts

```diff
@@ -0,0 +1,807 @@
+      expect(res1.body).toMatchObject<DisableDomainResponse>({
+        success: false,
+        version: res1.body.version,
+        error: WarningMessage.INVALID_INPUT,
+      })
+
+      const badRequest2 = ''
+
+      const res2 = await request(app).post(CombinerEndpoint.DISABLE_DOMAIN).send(badRequest2)
+
+      expect(res2.status).toBe(400)
+      expect(res2.body).toMatchObject<DisableDomainResponse>({
+        success: false,
+        version: res2.body.version,
+        error: WarningMessage.INVALID_INPUT,
+      })
+    })
+
+    it('Should respond with 401 on failed auth', async () => {
+      // Create a manipulated request, which will have a bad signature.
+      const badRequest = await disableRequest()
+      badRequest.domain.salt = defined('badSalt')
+
+      const res = await request(app).post(CombinerEndpoint.DISABLE_DOMAIN).send(badRequest)
+
+      expect(res.status).toBe(401)
+      expect(res.body).toMatchObject<DisableDomainResponse>({
+        success: false,
+        version: res.body.version,
+        error: WarningMessage.UNAUTHENTICATED_USER,
+      })
+    })
+
+    // TODO: test this with signers disabled too
+    it('Should respond with 503 on disabled api', async () => {
+      const configWithApiDisabled = combinerConfig
+      configWithApiDisabled.domains.enabled = false
+      const appWithApiDisabled = startCombiner(configWithApiDisabled)
+
+      const req = await disableRequest()
+
+      const res = await request(appWithApiDisabled).post(CombinerEndpoint.DISABLE_DOMAIN).send(req)
+
+      expect(res.status).toBe(503)
+      expect(res.body).toMatchObject<DisableDomainResponse>({
+        success: false,
+        version: res.body.version,
+        error: WarningMessage.API_UNAVAILABLE,
+      })
+    })
+  })
+
+  describe(`${CombinerEndpoint.DOMAIN_QUOTA_STATUS}`, () => {
+    it('Should respond with 200 on valid request', async () => {
+      const res = await request(app)
+        .get(CombinerEndpoint.DOMAIN_QUOTA_STATUS)
+        .send(await quotaRequest())
+
+      expect(res.status).toBe(200)
+      expect(res.body).toMatchObject<DomainQuotaStatusResponse>({
+        success: true,
+        version: res.body.version,
+        status: { disabled: false, counter: 0, timer: 0, now: res.body.status.now },
+      })
+    })
+
+    it('Should respond with 200 on repeated valid requests', async () => {
+      const req = await quotaRequest()
+      const res1 = await request(app).get(CombinerEndpoint.DOMAIN_QUOTA_STATUS).send(req)
+      expect(res1.status).toBe(200)
+      expect(res1.body).toMatchObject<DomainQuotaStatusResponse>({
+        success: true,
+        version: res1.body.version,
+        status: { disabled: false, counter: 0, timer: 0, now: res1.body.status.now },
+      })
+
+      const res2 = await request(app).get(CombinerEndpoint.DOMAIN_QUOTA_STATUS).send(req)
+      expect(res2.status).toBe(200)
+      expect(res2.body).toMatchObject<DomainQuotaStatusResponse>(res1.body)
```

Comment:
I think the `res1.body.status.now` field is sometimes causing flakiness -- maybe it makes sense to extract out the expected response and then setting the second check to res2.body.status.now?